### ISSUE TITLE
Create Release fails -- "(refusing to allow a GitHub App to create or update workflow `.github/workflows/AddExistingAppOrTestApp.yaml` without `workflows` permission)"

### ISSUE THREAD

**Issue by buzzwick** [microsoft/AL-Go#562]
Moving on to the next new error--

Clearly this is a permission setting somewhere, but I don't have enough information/experience to know where:

Run $ErrorActionPreference = "STOP"
  $ErrorActionPreference = "STOP"
  Set-StrictMode -version [2](https://github.com/CAI-Software-Inc/CAI-Software-Precious-Metals-Solution/actions/runs/5188704722/jobs/9352746553#step:3:2).0
  git checkout -b release/21.8.94
  git config user.name buzzwick
  git config user.email buzzwick@users.noreply.github.com
  git commit --allow-empty -m "Release branch release/21.8.94"
  git push origin release/21.8.94
  shell: C:\Windows\System[3](https://github.com/CAI-Software-Inc/CAI-Software-Precious-Metals-Solution/actions/runs/5188704722/jobs/9352746553#step:3:3)2\WindowsPowerShell\v1.0\powershell.EXE -command ". '{0}'"
  env:
    ALGoOrgSettings: {"useCompilerFolder": true,"doNotPublishApps": true,"artifact": "https://bcartifacts.azureedge.net/sandbox/22.0.5[4](https://github.com/CAI-Software-Inc/CAI-Software-Precious-Metals-Solution/actions/runs/5188704722/jobs/9352746553#step:3:4)1[5](https://github.com/CAI-Software-Inc/CAI-Software-Precious-Metals-Solution/actions/runs/5188704722/jobs/9352746553#step:3:5)[7](https://github.com/CAI-Software-Inc/CAI-Software-Precious-Metals-Solution/actions/runs/5188704722/jobs/9352746553#step:3:7).55210/us"}
    ALGoRepoSettings: 
Switched to a new branch 'release/21.[8](https://github.com/CAI-Software-Inc/CAI-Software-Precious-Metals-Solution/actions/runs/5188704722/jobs/9352746553#step:3:8).[9](https://github.com/CAI-Software-Inc/CAI-Software-Precious-Metals-Solution/actions/runs/5188704722/jobs/9352746553#step:3:9)4'
[release/21.8.94 e7e775d] Release branch release/21.8.94
To https://github.com/CAI-Software-Inc/CAI-Software-Precious-Metals-Solution
 ! [remote rejected] release/21.8.94 -> release/21.8.94 (refusing to allow a GitHub App to create or update workflow `.github/workflows/AddExistingAppOrTestApp.yaml` without `workflows` permission)
error: failed to push some refs to 'https://github.com/CAI-Software-Inc/CAI-Software-Precious-Metals-Solution'
Error: Process completed with exit code 1.

**Comment by freddydk** [comment]
It looks like you are creating a release branch that already exists???

**Comment by buzzwick** [comment]
Actually, yes. I failed to notice that even though the previous workflow run had failed on the creation of the release BRANCH, it had in fact successfully created the release itself.

I deleted the existing release because we need to create a new one with the same versioning and tags, etc. (trying to line everything up with our AppSource listing submission; it’s complicated).

After deleting the existing release, Create Release ran without error. Onward.
